ci-operator/step-registry/dpf/hypervisor/sanity-existing/dpf-hypervisor-sanity-existing-commands.sh

```
@@ -66,22 +65,22 @@ fi
-scp ${SSH_OPTS} root@${REMOTE_HOST}:${LAST_OPENSHIFT_DPF}/kubeconfig.doca8 /tmp
+scp ${SSH_OPTS} root@${REMOTE_HOST}:${LAST_OPENSHIFT_DPF}/kubeconfig.${CLUSTER_NAME} /tmp
 
 ls -ltr /tmp
-cp /tmp/kubeconfig.doca8 /tmp/kubeconfig.doca8_ORIG
+cp /tmp/kubeconfig.${CLUSTER_NAME} /tmp/kubeconfig.${CLUSTER_NAME}_ORIG
 
-echo " Substitute the hypervisor domain name 'api.doca8.nvidia.eng.rdu2.dc.redhat.com' for cluster api ip address '${CLUSTER_API_IP}'"
+echo " Substitute the hypervisor domain name 'api.${CLUSTER_NAME}.nvidia.eng.rdu2.dc.redhat.com' for cluster api ip address '${CLUSTER_API_IP}'"
 
-sed -i "s|server: https://api.doca8.nvidia.eng.rdu2.dc.redhat.com:6443|server: https://${CLUSTER_API_IP}:6443|" /tmp/kubeconfig.doca8
+sed -i "s|server: https://api.${CLUSTER_NAME}.nvidia.eng.rdu2.dc.redhat.com:6443|server: https://${CLUSTER_API_IP}:6443|" /tmp/kubeconfig.${CLUSTER_NAME}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)